README.md

```diff
@@ -14,6 +23,35 @@ And then execute:
 
     $ bundle
 
-## Usage
+Or install it yourself as:
+
+    $ gem install govuk_navigation_helpers
+
+
+### Usage
+
+Query the content store using gds-api-adapters:
+
+```
+TODO
+```
+
+Get the data you need for your component, eg
+
+```
+# TODO
+```
+
+Pass to slimmer:
+
+```
+TODO
+```
+
+### Running the test suite
+
+`bundle exec rake`
+
+## Licence
 
-TBD.
+[MIT License](LICENCE.txt)
```

Comment:
I think licenCe would be correct here and licenSe is the verb (stupid english language)

I'm updating the file to match our other libraries.